sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 - Added support for the Opinion Mining feature. To use this feature, you need to make sure you are using the 
 service's v3.1-preview.1 and above API. To get this support pass `includeOpinionMining` as `true` in 
 `AnalyzeSentimentOptions` when calling the sentiment analysis endpoints.
+- In API version v3.1-preview.2 and up, the redacted text of the document is returned on the result 
```

Comment:
Just realized, PII is new to the customer and it comes back only in our 5.1.0-preview.1 version, so we shouldn't include this line int he changelog as for the user, there wasn't any `PiiEntityCollection` property before

Reply:
I see. I can remove it.